packages/iov-core/src/multichainsigner.spec.ts

```diff
@@ -173,6 +174,80 @@ describe("MultiChainSigner", () => {
 
       signer.shutdown();
     });
+
+    it("can add two chains and send on both of them", async () => {
+      pendingWithoutBnsd();
+      pendingWithoutEthereum();
+
+      const profile = new UserProfile();
+      const signer = new MultiChainSigner(profile);
+      expect(signer.chainIds().length).toEqual(0);
+
+      await signer.addChain(bnsConnector(bnsdTendermintUrl));
+      await signer.addChain(ethereumConnector(httpEthereumUrl, undefined));
```

Comment:
This is a bit strange. It would be more idiomatic if the type was `null`able than `undefined`able.

Reply:
Oh the debate over the two versions of null in JS are probably going to live longer than us :) In all the places where we have TS support, I guess both ways are probably equally good.

I have no good reasons for one or the other and use `undefined` by default

Reply:
The rule I go by is something like: `undefined` is where you don't have knowledge of the value, `null` is where you know there is no value. I find it a helpful distinction, didn't realise it was controversial!